### ISSUE TITLE
Problem to show loaded images in ListView

### ISSUE THREAD

**Issue by omid-nazifi** [square/picasso#1209]
Hello
I've created a Gallery and used Picasso in getview() for loading images. Actually I've used Picasso in adapter before and there wasn't any problem. But now when I call resize method, I see following issues:
Picasso.with(getContext())
                .load(imageUrls.get(position))
                .resize(0, targetHeight)
                .into(viewHolder.imgGallery);
here first of all don't show images but when I come back and agian come into activity show loaded images.

How can I do solve it?

**Comment by JakeWharton** [comment]
You haven't provided enough information for anyone to really diagnose anything. Is targetHeight non-zero when this code runs?

**Comment by omid-nazifi** [comment]
@JakeWharton  As a matter of fact, I want resize images that loaded by Picasso library. On the other hand these images should load in ListView. So I code the below code but images don't show in ImageView and I don't know why!

Picasso.with(getContext())
.load(imageUrls.get(position))
.resize(0, targetHeight)
.into(viewHolder.imgGallery);

I've used these code before.
No targetHeight isn't zero. even I set a constant "200" to it.
